app/src/pages/inside/settingsPage/settingTabs/demoDataTab/demoDataTabForm/demoDataTabForm.scss

```diff
@@ -0,0 +1,46 @@
+.input-wrapper {
```

Comment:
Please add class `demo-data-form` and move corresponding classes inside (generate-buitton, and the class for input).
Since I suggest using `className` property for custom styling inputs, there may be a problem with specificity. It works now (I checked), but there might be a problem in future (for example, if style definitions order change in the resulting HTML file).